### PR TITLE
Fix for when surname is null/undefined

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

File: go/vumitools/contact/models.py [hunk 1]
```diff
@@ -136,12 +136,13 @@ def filter_contacts_on_surname(self, letter, group=None):
...
         js_function = """function(value, keyData, arg){
             var data = Riak.mapValuesJson(value)[0];
-            var surname = data.surname.toLowerCase();
-            if(surname && surname[0] === arg){
-                return [[value.key, value[0]]];
-            } else {
-                return [];
+            if(data.surname) {
+                var surname = data.surname.toLowerCase();
+                if(surname && surname[0] === arg){
```
Comment (jerith) [inline go/vumitools/contact/models.py]: Do you need to check for the existence of `surname` here now that you're checking `data.surname` above?

Comment (smn) [inline go/vumitools/contact/models.py]: surname can still be an empty string.

Comment (smn) [inline go/vumitools/contact/models.py]: Ah right. Removing.

**Comment by peterp** [comment]
+1

**Pull request merged**